packages/zora/src/Zora.ts

```diff
@@ -97,8 +100,15 @@ export const getMintIntent = async (
 ): Promise<TransactionRequest> => {
   const { contractAddress, tokenId, amount, recipient } = mint
   let data
-  if (tokenId !== 0) {
-    const mintArgs = [recipient, tokenId, amount, zeroHash]
+  if (tokenId !== null && tokenId !== undefined) {
+    const msgArgs = '0x' + recipient.slice(2).padStart(64, '0')
```

Comment:
[nit] I think we can use viem's `pad` with size = 64 to do this

https://viem.sh/docs/utilities/pad#size

Reply:
Cool - will do